docs/userguide/modules/exposing-binding-data/pages/supported-operators.adoc

```diff
@@ -0,0 +1,26 @@
+= Known bindable operator-backed services
```

Comment:
we have this content in `README.md` - why this needs to added to the userguide, i.e. it seems that this change is out of the PR focus?

Reply:
I'm cross referencing his from the quickstart so I wanted to keep the reading flow to stay in the userguide... eventually, to 
move this part out from `README.md` into the userguide and link it from the readme.

Reply:
initially we wanted to make that out-of-the-box support a temp thing, hoping to gradually remove it with SBO adoption. Therefore we did not include it in the user guide.

Reply:
ok, got it - will remove it